packages/uninstaller/src/aws.ts

```diff
@@ -71,6 +71,10 @@ export function* destroyAWSInfra(): Generator<
     clusterName: destroyConfig.clusterName,
     suffix: "loki"
   });
+  const lokiConfigBucketName = getBucketName({
+    clusterName: destroyConfig.clusterName,
+    suffix: "loki-config"
+  });
```

Comment:
ok, good, but we after all forget to trigger the lifecycle-based teardown below where we have

```
  taskGroup4.push(
    yield fork([
      new S3BucketRes(destroyConfig.clusterName, lokiBucketName),
      "teardown"
    ])
  );
```